### PR TITLE
feat: [SRM-13463]: Common assign component fixes

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary

1. Renamed SelectHealthSourceServices to AssignQuery
2. Default value for service instance 
3. Template flow for Assign component
4. Corrected margin with in Assign component

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/RiskProfile.module.scss [hunk 1]
```diff
@@ -6,6 +6,13 @@
  */
 
 .main {
+  :global {
+    p[class*='font-variation-card-title'],
+    p[class*='font-variation-body'] {
+      font-size: 12px !important;
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/RiskProfile.module.scss:12]: can we add `--font-size-small` for this?

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/components/RiskProfileCategeory/RiskProfileCategeory.module.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.riskProfileRadioGroup {
+  div[data-tooltip-id='riskProfileBaselineDeviation'] > p {
+    font-size: 12px !important;
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/components/RiskProfileCategeory/RiskProfileCategeory.module.scss:10]: same here

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/RiskProfile.module.scss (hunk 1 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/RiskProfile.module.scss]: done

File: src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/components/RiskProfileCategeory/RiskProfileCategeory.module.scss (hunk 2 above)
Comment (deepesh-ui) [reply src/modules/85-cv/pages/health-source/connectors/CommonHealthSource/components/CustomMetricForm/components/Assign/components/RiskProfile/components/RiskProfileCategeory/RiskProfileCategeory.module.scss]: done

**Review by harshilbits — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**